src/common/end-of-chapter-exercise.Rmd

```diff
@@ -5,7 +5,7 @@ There is a set of exercises available for this chapter!
 
 **Not sure how to access and work on the exercise Rmd files? **
 
--   Refer to Chapter 3.2 for step-by-step instructions on accessing the exercises and working within the UofT JupyterHub's RStudio environment.
+-   Refer to Chapter 4.2 for step-by-step instructions on accessing the exercises and working within the UofT JupyterHub's RStudio environment.
```

Comment:
Please actually change this reference to a hyperlink (linking to the specific section)